soc/intel/intel_adsp/ace/asm_memory_management.h

```diff
@@ -38,7 +38,7 @@
 	movi \az, 0x2f5
 	slli \az, \az, 0xb
 	/* 8 * (\segment_index << 5) == (\segment_index << 5) << 3 == \segment_index << 8 */
-	addmi \az, \az, \segment_index
+	addmi \az, \az, \segment_index << 8
```

Comment:
For ACE MAX_MEMORY_SEGMENTS == 1, so segment_index is always 0. And so this shift cannot affect the problem.

Reply:
@serhiy-katsyuba-intel  Ack, the patch affects via LSPGCTL_HIGH which is also passed to addmi. So this change is to just correct all uses of addmi. @lyakh to confirm, but that's how I understood it

Reply:
correct. This is just a correctness fix, the real fix is the `LSPGCTL_HIGH` part